src/workflows-app/utils/submission-utils.ts

```diff
@@ -45,6 +46,26 @@ export type OutputTableData = {
   variable: string;
 };
 
+export type WorkflowMetadata = {
```

Comment:
Exiting to have this type! This is a big step towards the WX team reaching 100% typescript.

Reply:
My dream is to have each key get its data from a custom accessor that makes a request only for its specific value.